CaImAn/caiman_main.py

```diff
@@ -165,6 +173,72 @@ def cnmf(n_processes, dview, fname_new, caiman_params):
     return cnm.estimates, images
 
 
+def cross_register(
+    A: np.ndarray,
+    C: np.ndarray,
+    shape,
+    caiman_parameters
+) -> List[int]:
+
+    if not caiman_parameters.params_cross_reg:
+        return
+    print(cm_defs.Messages.CROSS_REGISTRATING , flush=True)
+
+    # Load reference images (AC_ref) and ROI footprints (A_ref)
+    ref_filepath = caiman_parameters.params_cross_reg["fname"]
+    ref_rois_path  = caiman_parameters.params_cross_reg["h5path_roi"]
+    ref_images_path = utils.clean_path(caiman_parameters.params_cross_reg["h5path_images"])
+    with h5py.File(ref_filepath, 'r') as file_:
+        AC_ref = np.array(file_[f"{ref_images_path}/{caiman_parameters.dataname}"])
+        A_ref, roi_ids_ref = get_footprints(file_, ref_rois_path, AC_ref.shape)
+
+    # Concatenate max proj of images (AC and AC_ref)
+    shape = (shape[1], shape[2], shape[0]);
```

Comment:
do we have to do it on a separate line or can we just write it inside the reshape on the line 197?